ai-platform-unified/notebooks/unofficial/hyperparameter_tuning/distributed-hyperparameter-tuning.ipynb

```diff
@@ -0,0 +1,879 @@
+{
```

Comment:
<p>Change to present tense.</p><p>After your container has been pushed to Google Container Registry -&gt;</p><p>Once your container is pushed to Google Container Registry</p><p> </p><p>You define the following specs -&gt; You define the following specifications</p><p> </p><p>that has two&nbsp;<code>NVIDIA_TESLA_T4</code>&nbsp;GPUs. -&gt; with two&nbsp;<code>NVIDIA_TESLA_T4</code>&nbsp;GPUs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Done